ui/packages/catalog-ui-search/src/main/webapp/component/query-add/query-add.view.js

```diff
@@ -162,6 +181,57 @@ module.exports = Marionette.LayoutView.extend({
                 }.bind(this));
         }
     },
+    getQueryAsQueryTemplate: function() {
+        let formParameters = this.queryContent.currentView.serializeTemplateParameters();
+        let filterTree = cql.simplify(formParameters.filterTree || {});
+        let filterSettings = formParameters.filterSettings || {};
+        let formModel = this.model.get('associatedFormModel') || new SearchFormModel();
+        if (filterTree.filters && filterTree.filters.length === 1) {
+            filterTree = filterTree.filters[0];
+        }
+        filterSettings.sorts = filterSettings.sorts.filter(sort => sort.attribute && sort.direction)
+            .map(sort => sort.attribute + ',' + sort.direction);
+        let filterTemplate = {
+            filterTemplate: filterTree,
+            accessIndividuals: formModel.get('accessIndividuals'),
+            accessGroups: formModel.get('accessGroups'),
+            creator: formModel.get('createdBy'),
+            id: formModel.get('id'),
+            title: this.model.get('title'),
+            description: formModel.get('description'),
+            created: formModel.get('createdOn'),
+            owner: formModel.get('owner'),
+            querySettings: filterSettings
+        }
+        return filterTemplate;
+    },
+    saveTemplateToBackend: function() {
+        let loadingView = new LoadingView();
+        $.ajax({
+            url: '/search/catalog/internal/forms/query',
+            data: JSON.stringify(this.getQueryAsQueryTemplate()),
+            method: 'PUT',
+            contentType: 'application/json',
+            customErrorHandling: true
+        })
+        .done((data, textStatus, jqxhr) => {
```

Comment:
Do we need to confirm success?

Reply:
`done` is a drop-in replacement for `success`
```
 jqXHR.done(function( data, textStatus, jqXHR ) {});
An alternative construct to the success callback option, refer to deferred.done() for implementation details.
```
https://api.jquery.com/jquery.ajax/

Reply:
Fixed `announcement` component to properly accept a timeout and use 2.5s instead of 5s